src/middlewared/middlewared/schema.py

```diff
@@ -181,14 +181,20 @@ def __init__(self, *args, **kwargs):
         super(IPAddr, self).__init__(*args, **kwargs)
 
     def validate(self, value):
+        verrors = ValidationErrors()
+
         if value:
             try:
                 if self.cidr:
-                    ipaddress.ip_interface(value)
+                    ipaddress.ip_network(value, strict=True)
```

Comment:
I have the impression this will break other things using CIDR?

We want host bit sets for things that accept cidr ip address, e.g. 192.168.3.2/24, where as here 192.168.3.0/24 will fail

Reply:
`strict=True` is default, I should remove that

What is the point of having host bits set?

Also `ipaddress.ip_interface` will accept just IP address which was my primary concert

Reply:
For backward compatibility. We already allow that and backend (mountd) does not enforce the host bits to not be set.

Reply:
Also, we need for CIDR specification, not just for network